common/lib/xmodule/xmodule/js/src/video/02_html5_video.js

```diff
@@ -61,17 +61,6 @@ function(_) {
         function Player(el, config) {
             var errorMessage, lastSource, sourceList;
 
-            // A simple test to see that the 'config' is a normal object.
-            if ($.isPlainObject(config) === false) {
-                return;
-            }
-
-            // We should have at least one video source. Otherwise there is no
-            // point to continue.
-            if (!config.videoSources && !config.videoSources.length) {
-                return;
-            }
```

Comment:
What about if we don't have video source after removing this code ? Control will never come to this place ?